lib/default_assign/hooks/default_assign_issues_hooks.rb

```diff
@@ -7,7 +7,7 @@ def view_issues_form_details_top(context = {})
     interactive_assignment =
       Setting.plugin_redmine_default_assign['interactive_assignment'] || 'true'
     interactive_assignment = (interactive_assignment == 'true')
-    return  if not interactive_assignment
+    return  if !interactive_assignment || Setting.plugin_redmine_default_assign[:override_category_assignee] != 'true'
```

Comment:
Would you mind splitting this into two separate return statements, to avoid the long line?